ui/app/AppLayouts/Chat/panels/UserListPanel.qml

```diff
@@ -114,8 +105,7 @@ Item {
             section.delegate: (root.width > 58) ? sectionDelegateComponent : null
             delegate: StatusMemberListItem {
                 width: ListView.view.width
-                nickName: model.localNickname
-                userName: ProfileUtils.displayName("", model.ensName, model.displayName, model.alias)
+                userName: model.preferredDisplayName
```

Comment:
Pls leave this part alone :) The StatusMemberListItem does its own composition, depending on both `nickName` and `userName`

Reply:
I started working on the `StatusMemberListItem`, I can take care of this in a followup PR